myia/parser.py

```diff
@@ -208,12 +210,15 @@ def make_location(self, node: ast.AST) -> Location:
             # we basically just pass through them.
             return None  # pragma: no cover
 
-    def parse(self) -> Graph:
+    def parse(self, clean=True) -> Graph:
```

Comment:
I can, if I decide to keep it. This branch is in a rough WIP state and this is a bit of a hack, so it might not remain. But basically it determines whether dead code elimination is triggered or not, and I left an option not to trigger it so that DCE could be tested.

Reply:
OK.